### ISSUE TITLE
Date order

### ISSUE THREAD

**Issue by ghost** [ddumont/nillablog#1]
Is there a way to keep them in chronological order of the post and not change when someone publish a comment?

**Comment by ddumont** [comment]
I think it does that but only in the blog view...   Does it not?

**Comment by ghost** [comment]
no, in the blog view the order change with a new comment :(, I'm using vanilla 2.3.1 and this is the only issue

**Comment by ddumont** [comment]
https://github.com/ddumont/nillablog/blob/master/src/main/php/class.nillablog.plugin.php#L164

I guess they broke the way I was doing this here in newer versions.
I honestly haven't touched vanilla in a while.  If you want, try posting on
the forum to see if someone can send me a patch and i'll make a new release.

**Comment by ghost** [comment]
Hi, I found a solution in the vanilla forum that has worked well for me - [https://open.vanillaforums.com/discussion/comment/215807/#Comment_215807](https://open.vanillaforums.com/discussion/comment/215807/#Comment_215807)

**Comment by ddumont** [comment]
Can you share it?   I'll update the code.
